src/Promitor.Integrations.Sinks.Statsd/Configuration/StatsdFormatterTypes.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Promitor.Integrations.Sinks.Statsd.Configuration
+{
+    public class StatsdFormatterTypes
```

Comment:
Can't we use an enumeration instead?

Reply:
I started with enumeration but then we have numeric values in the helm chart, such as FormmaterType: 1
also it makes the deserialization from number to enum a little bit trickier

Reply:
We can use enum which is serialized as string which is better though